Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -65,22 +69,31 @@ There are usually two release engineers working at any given time. It goes witho
 **Phase 5: Closure**
 <br/>	*It starts after the app is accepted by Apple and final internal approval*
 
-1. Send the build to Testflight Beta (external testing). Select the `External Testers` group.
+1. Send the build to TestFlight Beta (external testing). Select the `External Testers` group.
 1. Press `Release this version` in App Store Connect
-1. Tag the release commit and create a GitHub release. Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
+1. Create tag(s) on the release commit and create GitHub releases for each app
+  - Make sure you create one tag and one GitHub release for each app released on the AppStore (eg. one for Babylon and one for Telus, in the most common case of joint releases of those two – even if they end up pointing to the same commit)
+  - Use the `{appname}/{version}` format for the tag name (eg. `babylon/3.2.0` or `telus/3.2.0`)
+  - Set the body of the GitHub release to the content of the Release Notes for the app
+  - Attach the binary as an artefact to the GitHub release (if you're using the automated release command, you can find the binary in the Artifacts top section in the CI build).
 1. Merge the changes back to develop.
 1. Update the [release calendar](#release-calendar)
 
-### 3. SDK Release
+## 3. SDK Release
 
 1. Ask SDK team (#sdk_squad) about the SDK version number.
-2. Follow the [Internal SDK Release Process](https://engineering.ops.babylontech.co.uk/docs/cicd-deployments/#mobile-sdk-releases-ios-android) to open a CR ticket on the CRP board.
-3. Cut a release branch for the SDK from the app release branch (eg. `sdk/0.5.0`)
-4. Create PR and udpate the SDK changelog `SDK/CHANGELOG.md` to add the release version and date
-5. Trigger a hockey build from that branch using its command (eg. `/fastlane distribute_sdk_v2 version:0.5.0 branch:release/sdk/0.5.0`) in `#ios-build`.
-6. Update the Sample app to point to the latest SDK release
+1. Cut a release branch for the SDK from the app release branch, using the `sdk/{version}` naming convention (eg. `sdk/0.5.0`)
+1. Create a CRP ticket by triggering its command (eg. `/crp sdk branch:0.5.0`) in Slack
+  - This will create the CRP ticket for the SDK, only including in the CHANGELOG field of the CRP the commits messages containing `[SDK-xxx]` or `#SDK` – filtering out the other commits (that are considered app-only changes if not containing those tags)
```

Comment:
Note: There are plans to enforce that convention via Danger – so that any PR which contains changes to any fiiles in the `SDK/` folder will be required to have the `#SDK` tag in its title to ensure the squashed commit will be listed in the SDK CRP ticket.